client/src/index.js

```diff
@@ -71,11 +87,20 @@ function App() {
 
 const $root = document.getElementById("root");
 
-render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
-  $root
-);
+async function main() {
```

Comment:
An amazing way to get the user of the backend.

Reply:
Hey, this looks like my async await speech 👀